frontend/js/src/home/Homepage.tsx

```diff
@@ -248,11 +293,196 @@ function HomePage() {
+                to="https://en.wikipedia.org/wiki/Open_source"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Wikipedia
+              </Link>
+            </p>
+            <p>
+              The ListenBrainz website, the MusicBrainz database, and the music
+              and listen data that our users contribute is all open source. This
+              means that anyone can view the code, contribute improvements and
+              new features, and copy and modify it for their own use.
+            </p>
+            <p>
+              Thousands of people contribute code or data to ListenBrainz and
+              its sister projects for no monetary return. Contributors do this
+              to further humanity&apos;s global store of knowledge, and to
+              safeguard and share data about the artists and music that we love.
+            </p>
+            <p>
+              Data wants to be free, and we believe that by sharing what we
+              have, rather than jealously guarding it for the benefit of a few,
+              everyone can benefit.
+            </p>
+            <Link
+              to="https://github.com/orgs/metabrainz/repositories"
+              className="info-link"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <FontAwesomeIcon icon={faAnglesRight} size="sm" /> GitHub
+              repositories
+            </Link>
+          </div>
+
+          <div className="info-column">
+            <div className="info-icon">
+              <img
+                src="/static/img/homepage/LB-Data-Provider-Info.png"
+                alt="Data provider"
+              />
+            </div>
+            <h2>Data provider</h2>
+            <p>
+              The MetaBrainz core mission is to curate and maintain public
+              datasets that anyone can download and use. Some of the world’s
+              biggest platforms, such as Google and Amazon, use our data, as
+              well as small-scale developers and simply curious individuals.
+            </p>
+            <p>
+              ListenBrainz and its sister sites are part of this mission - the
+              music data that our users submit to ListenBrainz, and the
+              MusicBrainz database that powers it, is available for download. We
+              want to help <strong>you</strong> build products and tools.
+            </p>
+            <p>
+              These datasets include the ListenBrainz PostgreSQL Data Dumps,
+              which contains detailed data about ~1 billion listens, and can be
+              used to create and study music consumption patterns. Our datasets
+              are AI Ready, perfect for training large language models for
+              music-based tasks.
+            </p>
+            <p>
+              We ask{" "}
+              <Link
+                to="https://metabrainz.org/supporters/account-type"
```

Comment:
Sorry, this page is better, my bad.

```suggestion
                to="https://metabrainz.org/supporters"
```